### ISSUE TITLE
vmalert not working with vmauth

### ISSUE THREAD

**Issue by rongfengliang** [VictoriaMetrics/VictoriaMetrics#514]
**Describe the bug**
vmalert  with vmauth error
**To Reproduce**

links https://github.com/rongfengliang/vmalert-vmauth

**Screenshots**

```code
vmalert_1       | 2020-05-22T09:07:01.177Z      error   VictoriaMetrics/app/vmalert/group.go:148        failed to execute rule "TestGroup"."Conns": failed to execute query "sum(vm_tcplistener_conns) by(instance) > 1": error parsing metrics for http://vmauth:8427/api/v1/query?query=sum%28vm_tcplistener_conns%29+by%28instance%29+%3E+1:invalid character '\x1f' looking for beginning of value
vmalert_1       | 2020-05-22T09:07:01.179Z      error   VictoriaMetrics/app/vmalert/group.go:148        failed to execute rule "TestGroup"."ExampleAlertAlwaysFiring": failed to execute query "sum by(job) (up == 1)": error parsing metrics for http://vmauth:8427/api/v1/query?query=sum+by%28job%29+%28up+%3D%3D+1%29:invalid character '\x1f' looking for beginning of value
```

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
docker run --rm -it victoriametrics/vmalert -h
vmalert-20200521-152717-tags-v1.35.6-cluster-0-gdcbdc009f
```

**Comment by rongfengliang** [comment]
this cause by default vmauth response is gzip disable it

**Comment by valyala** [comment]
@rongfengliang , this issue must be fixed in the commit b3fcd726e38830a0834e059656d89d87469a2d9b . Could you try building `vmauth` from this commit according to [these docs](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmauth/README.md#how-to-build-from-sources) and verify whether the issue is fixed there?

Re-opening the issue in order to verify whether it has been fixed.

**Comment by rongfengliang** [comment]
@valyala it works,thanks

**Comment by valyala** [comment]
The bugfix has been included in `vmalert` and `vmauth` v1.36.0 - see [the release notes](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) for details.

Closing this issue as fixed.
